clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/vmimage/view/AzureVMImageLookupController.groovy

```diff
@@ -61,15 +62,31 @@ class AzureVMImageLookupController {
   List<AzureNamedImage> getVMImage(@PathVariable String account, @PathVariable String region, @PathVariable String imageId) {
```

Comment:
Current plan is to use jenkins/igor -> echo to trigger the pipeline; we don't plan to use "bakery" at the moment. BTW I just start looking into these, setting them up and figure out the exact workflow from the AWS side so I'm still ramping up.